File: ui/src/models/ong.ts
export interface Ong {
  id: number
  name: string
  uin: string
  address: string
  lat: number
  long: number
}

export const defaultOng = (): Ong => {
  return {
    id: 0,
    name: '',
    uin: '',
    address: '',
    lat: 0,
    long: 0,
  }
}

File: server/Models/Ong.cs
using System.ComponentModel.DataAnnotations;

namespace apifmu.Models
{
    public class Ong
    {
        [Key]
        public int Id { get; set; }

        public string Uin { get; set; }

        public string Name { get; set; }

        public string Address { get; set; }

        public double Lat { get; set; }

        public double Long { get; set; }
    }
}
